src/main.js

```diff
@@ -52,17 +54,22 @@ let isNeptuneIAMAuth = false;
 let createUpdatePipeline = false;
 let createUpdatePipelineName = '';
 let createUpdatePipelineEndpoint = '';
+let createUpdatePipelineEndpointRO = '';
+let createUpdatePipelineDualLambdas = false;
 let createUpdatePipelineRegion = 'us-east-1';
 let createUpdatePipelineNeptuneDatabaseName = '';
 let removePipelineName = '';
 let inputCDKpipeline = false;
 let inputCDKpipelineName = '';
 let inputCDKpipelineEnpoint = '';
+let inputCDKpipelineEnpointRO = '';
+let inputCDKpipelineDualLambdas = false;
```

Comment:
Andrea actually wanted us to remove the code for dual lambdas option as it is not finished. Can you please remove this and all references to it?

Reply:
Removed 👍